packages/pangea-node-sdk/src/utils/validation.ts

```diff
@@ -0,0 +1,92 @@
+export const schema = {
```

Comment:
What is this validation used on? Is there a reason we don't just use something like yup?

Reply:
This validation is used on validating the parameters passed in to each authn method. 

I went back and forth between using `yup` or `joi` (https://joi.dev/api/?v=17.7.0) for schema validation. TL;DR: instead of pulling in another dependency, I just needing something easy and super lightweight to do basic validation.

Decided not to use `joi` because of issues I've run into with it in the docusaurus plugins. Every instance of `joi` needs to be the exact same version, so if we start using `joi@17.7.0` for validation and someone using our SDK in their app has a different version, they will not be able to build/run their app.

Decided not to use `yup` because it also pulls in more dependencies, but if we're okay with that, then I could pull it in.

Ultimately, I wanted a super lightweight, quick and easy way to validate parameters passed into each method.

Reply:
Pulled validation out of this pull request to reduce scope.

I think we'll want to add validation at some point, but that should be it's own pull request instead of being bundled with AuthN